app/Http/Controllers/User/BookmarksController.php

```diff
@@ -154,8 +154,10 @@ public function store(Request $request)
         $book = Book::where('id', $request->book_id)->first();
         $request['book_id'] = $book->id;
         $request['bible_id'] = $request->dam_id ?? $request->bible_id;
-
-        $invalidBookmark = $this->validateBookmark();
+        $request['bible_id'] = getFilesetFromDamId($request['bible_id'], true);
```

Comment:
This code is responding to an v4 API call, which should already have the corrected bible_id because we've separately run the sync process to fix it. I don't think we need any logic in the Bookmarks/Notes/Highlights Controllers. Am I missing something?

Reply:
per Jose, this is to fix an existing error